### PR TITLE
FXIOS-868 ⁃ #7069: Top Sites and Open Tabs Widget

### PR DESCRIPTION
**Pull request by nbhasin2**
Fix issue with white border on quickLinks widget

(cherry picked from commit 75c6f5ae3d0d2b2802a482f064042bd4d3c1134c)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-868)

### CODE REVIEW

File: Client/Application/AppDelegate.swift [hunk 1]
```diff
@@ -376,6 +377,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         } else {
             syncOnDidEnterBackground(application: application)
         }
+        
+        tabManager.preserveTabs()
+
```
Comment (nbhasin2) [inline Client/Application/AppDelegate.swift:382]: nit: spacing

**Comment by nbhasin2** [comment]
@garvankeeley I commented out files for widgets that require refactoring but let me know if there is a better way to deprecate files, I can remove the files completely or comment them out.

**Review by garvankeeley — APPROVED** [review]
Let's land so we can get `main` healthy again. We have an xcode 12 build of A-S lib on there now.

**Comment by garvankeeley** [comment]
Sigh, tired of waiting for BB to finish building. Merging to master and running Bitrise to see if I can get iOS14 builds going there again.

**Pull request merged**